conslogging/delayed.go

```diff
@@ -0,0 +1,31 @@
+package conslogging
+
+import (
+	"fmt"
+)
+
+// DelayedLogger is a logger that queues up messages until Flush is called.
+type DelayedLogger struct {
```

Comment:
nit: usually writers that have a `Flush` method are called buffered. e.g. go's `bufio`.

```suggestion
type BufferedLogger struct {
```